plugin/diagnostics.py

```diff
@@ -322,7 +324,7 @@ def update_diagnostics_panel(window: sublime.Window):
             panel.settings().set("result_base_dir", base_dir)
 
             auto_open_panel = False
-            to_render = []
+            to_render = ""
```

Comment:
joining a list of strings is "significantly faster": https://stackoverflow.com/questions/3055477/how-slow-is-pythons-string-concatenation-vs-str-join